src/components/media-types/container.js

```diff
@@ -18,17 +18,43 @@ export const Container = ({ children = null, interactive }) => {
   })
 
   const toggleFullScreen = () => {
-    if (!document.fullscreenElement) {
-      document.documentElement.requestFullscreen()
+    // iPhone doesn't support Fullscreen API
+    // but we can toggle fullscreen class manually
+    const iPhone = /iPhone|iPod/.test(navigator.userAgent) && !window.MSStream
```

Comment:
Also, don't do it in the toggle Fullscreen function. that function shouldn't be called at all.

Do it at the beginning. and if its IOS just hide the button. leave the rest of the code

Reply:
i can move the check to the top, but fullscreen does "work" in that it will fill the screen, even if you still see the window chrome. thought this was better than completely disabling, or?

Reply:
![Screen Shot 2021-04-01 at 11 47 45 AM](https://user-images.githubusercontent.com/111979/113276426-1b1ac180-92e0-11eb-9216-44db640ec6a5.png)

Reply:
ok cleaned up as you suggested